components/certificate-mgt/org.wso2.carbon.certificate.mgt.core/pom.xml

```diff
@@ -163,6 +163,21 @@
             <artifactId>testng</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.h2database.wso2</groupId>
+            <artifactId>h2-database-engine</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-api-mockito</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-module-testng</artifactId>
+            <version>1.7.0</version>
```

Comment:
Please use the same version from the parent pom. If the dependency is not existing, then please define it from the parent pom and inherit it here.